eng/common/scripts/Update-DocsMsMetadata.ps1

```diff
@@ -225,10 +225,6 @@ foreach ($packageInfoLocation in $PackageInfoJsonLocations) {
   if ($ValidateDocsMsPackagesFn -and (Test-Path "Function:$ValidateDocsMsPackagesFn")) {
     Write-Host "Validating the package..."
     &$ValidateDocsMsPackagesFn -PackageInfo $packageInfo -PackageSourceOverride $PackageSourceOverride -DocValidationImageId $DocValidationImageId -DocRepoLocation $DocRepoLocation
-    if ($LASTEXITCODE) {
```

Comment:
Please be sure to file a follow-up issue which will figure out the correct way to handle validation errors.

Reply:
Found someone report an issue while I draft an issue to fix this.
https://github.com/Azure/azure-sdk-tools/issues/2669

Reply:
File an issue tracking the followup work
https://github.com/Azure/azure-sdk-tools/issues/2676